pkg/query-service/app/logs/v4/json_filter_test.go

```diff
@@ -183,6 +183,19 @@ var testGetJSONFilterData = []struct {
 		},
 		Filter: "lower(body) like lower('%message%') AND JSON_EXISTS(body, '$.\"message\"')",
 	},
+	{
+		Name: "exists",
+		FilterItem: v3.FilterItem{
+			Key: v3.AttributeKey{
+				Key:      "body.name",
+				DataType: "string",
+				IsJSON:   true,
+			},
+			Operator: "in",
+			Value:    []interface{}{"hello", "world"},
+		},
+		Filter: "lower(body) like lower('%name%') AND JSON_EXISTS(body, '$.\"name\"') AND JSON_VALUE(body, '$.\"name\"') IN ['hello','world']",
+	},
 }
```

Comment:
Can you add a couple of more tests with other data types?

Reply:
have added more tests